apps/internal/base/storage/partitioned_storage.go

```diff
@@ -91,6 +91,7 @@ func (m *PartitionedManager) Write(authParameters authority.AuthParams, tokenRes
 	environment := authParameters.AuthorityInfo.Host
 	realm := authParameters.AuthorityInfo.Tenant
 	clientID := authParameters.ClientID
+	resource := tokenResponse.Resource
```

Comment:
MSAL uses the **request** scopes for caching. This is because the client could ask for scopes "S1, S2, S3" and they could get "S1, S2, S3, S4, S5" or just "S1, S3".

Reply:
Updated that removed resource, now it is attached from request